### PR TITLE
default values isgri lower limit energy and default OSA version

### PR DESCRIPTION
**Pull request by burnout87**

### CODE REVIEW

**Comment by burnout87** [comment]
Isgri OSA, and energy lower limit:

image

JEMX OSA version:

image

**Comment by burnout87** [comment]
Some other defaults to set?

File: instruments/mmoda_isgri/mmoda_isgri.install [hunk 1]
```diff
@@ -45,12 +45,12 @@ function mmoda_isgri_install()
     'enabled' => TRUE,
     'name' => $instrument_info['name'],
     'title' => $instrument_info['title'],
-    'osa_version' => 'OSA11.1',
+    'osa_version' => 'OSA11.2',
```
Comment (volodymyrss) [inline instruments/mmoda_isgri/mmoda_isgri.install:48]: oh. and what are the possible options?

Comment (burnout87) [reply instruments/mmoda_isgri/mmoda_isgri.install:48]: ISGRI:
[https://github.com/oda-hub/mmoda-frontend-module/blob/ff1b8c0bb79cc03d67741fd3ee123f7405b82cab/instruments/mmoda_isgri/mmoda_isgri.inc#L42-L46](https://github.com/oda-hub/mmoda-frontend-module/blob/ff1b8c0bb79cc03d67741fd3ee123f7405b82cab/instruments/mmoda_isgri/mmoda_isgri.inc#L42-L46)

JEMX:
https://github.com/oda-hub/mmoda-frontend-module/blob/ff1b8c0bb79cc03d67741fd3ee123f7405b82cab/instruments/mmoda_jemx/mmoda_jemx.inc#L40-L43

Comment (burnout87) [reply instruments/mmoda_isgri/mmoda_isgri.install:48]: Some should be added?

**Review by ferrigno — APPROVED** [review]

File: instruments/mmoda_isgri/mmoda_isgri.install (hunk 1 above)
Comment (volodymyrss) [reply instruments/mmoda_isgri/mmoda_isgri.install:48]: no, this is good.

**Review by volodymyrss — APPROVED** [review]

**Pull request merged**